docs/plugin-vidarr.md

```diff
@@ -18,3 +18,22 @@ should be submitted to Vidarr. If null, worklow runs will always be submitted.
 If a number, in seconds, is given, then the action must have been
 (re-)generated by an olive in that window to be submitted to Vidarr. If that
 window is exceeded, they will be run as dry-run.
+
+The vidarr plugin provides an imput format: `vidarr_provenance` which 
+provides the analyis provenance data.
+To use, add to the configuration file:
+
+    {
+      "defaultMaxSubmissionDelay": null,
+      "url": "http://vidarr:8000",
+      "instance": "development",
+      "analysisTypes": ["FILE"],
+      "versionTypes": []
+    }
+
+The `"instance"` is used to determine workflow accession values.
+
+The `"analysisTypes"` specifies the type of analysis desired. 
```

Comment:
What are the available options and how do they affect the content of the input format?